pkg/booleanpolicy/violationmessages/printer/vuln.go

```diff
@@ -24,6 +24,12 @@ const (
     {{- if .FixedBy}}Fixable {{end}}{{.CVE}}{{if .CVSS}} (CVSS {{.CVSS}}){{end}}{{if .Severity}} (severity {{.Severity}}){{end}} found
```

Comment:
I don't think this should be an either or template - rather two separate templates one for cvss and one for nvd. The nvd score can be added, comma separated from the cvss score in the parens, in the same message.